Automattic/wp-calypso
# A4A > Marketplace: Implement VIP hosting top section

Closes https://github.com/Automattic/jetpack-genesis/issues/452

## Proposed Changes

* This PR implements VIP hosting top section for the new hosting page

Note:

We are keeping 4 logos per line on large screens as this looks better. I will check this with the design team too.

## Why are these changes being made?

* For the new hosting page to showcase the VIP hosting plan

## Testing Instructions

* Open the A4A live link
* Visit Marketplace - Hosting > Click the Enterprise tab 
* Verify

- UI looks good on all the screens, as shown below. Some changes will be made for the mobile view later in a different PR
- Click the Request a Demo button and verify that you are redirected to https://wpvip.com/contact/ in a new tab

<img width="1728" alt="Screenshot 2024-07-30 at 8 30 42 AM" src="https://github.com/user-attachments/assets/f0c65e88-9165-4b41-aef9-525a392ea7de">
<img width="1068" alt="Screenshot 2024-07-30 at 8 31 35 AM" src="https://github.com/user-attachments/assets/18209a64-0160-4185-9934-09d706bbd932">
<img width="573" alt="Screenshot 2024-07-30 at 8 37 35 AM" src="https://github.com/user-attachments/assets/f4357510-14e7-441b-9e23-34cb33b1fdc9">


## Pre-merge Checklist
